components/api/src/main/java/com/hotels/styx/api/LiveHttpResponse.java

```diff
@@ -82,12 +82,14 @@
     private final HttpResponseStatus status;
     private final HttpHeaders headers;
     private final ByteStream body;
+    private final LiveHttpRequest request;
```

Comment:
This field does not seem to be used anywhere (including via it's accessor method). We should leave it out.

Reply:
it can be accessed by the plugins

Reply:
Yash Vyas: this is actually used by the logging plugin like you said, and i can provide some context into why this is needed.